www/default/index.php

```diff
@@ -30,10 +68,46 @@
lSkD2q6thpE7LbmLFlwAMG4hy8AZwlJbYFGk2CRoZCYeIlXGCJs2SkJIXbo14DlT0EyhiPJKWCgUzs8WLAEUcG3MMDnPEML/CKbEgTU8yxJC17y9Te0ntv4YgBd1ispO3ghnHGUYN0k47DaGBHEulK2BoLnEhopfcT4rYSUIN5smEiE9KXJIMNdiSQgUQQ7HDAO5yIWV6CEFH+rwLRQCxohA5jRb5K1AEb7Egiv4mw54cagVL+TUFT0hF9LITnj47yT9UFk/LyiNvC6c+T+f/W66wPMiGlAzTIUMI8TpKi7EGFQZuAjCQGYYf7JJGxhK0k5bM9oS0sZpbMKkM9K0wmElbDAifJULMHCYM96Bnp06QXPZGope1NV1LeY3BM3mPz5v9bJpFZufoD2GCHu7iPE57iOV6SNZlLCMIMC9IkCwmpY9E2DE0E9AVC6WgzIoosHSwSTjiEwaFhxUP0/Z2rjPJY3Iol1eefvVX1/jebfAmoXaCVlDko0VZw2NTxIShWjI9ocrEvCLSvg/g2jwVi77yFUCNIhML2xYjjQ9h6IKsE5oG1JBaFg8KJO3yJ4WFHCHjn/fDrnxBQIMgRU9QVH3wFXuZ695o6TALCK2jXVapqg5dBfchwafIGJRMQXTwUUQ8FcyaobU6wShBTKDK1kbk2Pz7EHJTwLjhYBbGoK5HlAxHaD0rzMdfQnBJHQAK7fUuAKZMmjkGGFQJKGwGQCU8jDoEJIBAfd9iUhsOFABQIEtXJAcB07/8cNxFA9hHEcqEDCVAPAipB3C4IneuAmgsmREXJI4Agpqah5Gaf8uowHDyok+CI5zzOfZfsOYEwlbcvyILg945XiI/rSExkgE44svcTGINYRufhSECV6KhO64w9VQu8kKIghAw6zkSw4wDg97rXfFLnmq87suO3nUcSjGj/YKYw1VynOFhF6/UFdoGfVfIlIBMitJ8Bmo9obJY4vqMI8Gk7z9QmB6kKnghNlinL7KP/h1LqFHAETvY9TyeQIrXVN+WArE4mCkVE6sQWAiVutNGiU1KvENiKTmdChpxKjSDLKTfJfREiW2wEJhDAR00qRomNjf1aN3TCEbgKJ93nOw/euyQaRhvRCMTuJD14MeR6IJ4QyIa90K/i+HVjAOrtvHCiXRCo6R2BgbWBr3lPPx2gZf8ovOYpAH76lg8j9sf5fb0a4KBqtTiUAWSebvuAdu6ux9kOAjO0TM5DAaD1vxIuW82nhZhDPy5lJs1uqwdfBQ2IQ4QhKi5OMjpGRfoTl4mrlulkpGMTZgYD4mjtZGriWzdWCxYTR/cRDjDvAfgITyIOJY4wH1dk1KzrAO0bB0RkHAKVGUvRZVgbw7mmkKytDhs9EfZ1jxEIS/RPlo8ctiuWukwy4rlDBrNT64RPRkubY7vR49zo2AjC8AwwpJZIV8KOh61hwr3Sda+X6X8bvPDVeqv9YbHtum2cWZ97b741XrJ4LnzSPr353FX0vWCr7n8+MairZzchhYAvYDgNv1gNQi6/2kckHf3vUWjf7uUoteEgKUDDzNobDFvXWItjr9VvXee4IVvXG/PturVbmah4NGnTrFwlIg1pFUvTqZlFd2y1NvVK+bqkS8OXTkJKRUtNR3WsA3VWT8QFs9AcDdOvsmu2qK3P2ojo9qQ4eRC18sjH4DWatCpX5gxqhNNIGoM3KFWnUs+rOLZerTICaOKJluWgmULmNb3YVgNk443IYB/WUhWalKlc8/Ilb1BHWw01gMM89dZiqZZW5w1L/z+oqnZ/I3js1DNM7zSAGQYHNQZ8KLfOBGHj4ErDS8xtExDLIJEpm4yclp6BkZmFnS/TycXNxy9XnoBX5xUeBf+GadlthLf7UR39ffJtcVxPHBQ0TF9gxcNJkOi3E/92pMAjSO0bIhIyCioaOgYmFqgHGBxcaXj40gkIiYhlkMiUJZuUjJxCTo+wMDgCiUJjsDg8gUgiU6g0OoMJWGwOl8dvF1AoEkvqxcl6hlcoVWpNv9Q62ekNRj++2WK12fuqFXJ2cXVz9/D08vbx7Ved/13BECgMjkCi0D0kYHF4ApFEplBpdAaTxeZweXyBsMdEsUQqkyuUKrVGq9MbjCazxWqzO5wuN3cPTy9vH1JfyBQqjc5gstgcLo8vEIrEEqlMrlCq1BqtTm8AxGgyW6w2u8Ppcnu8Pr+33nnvg48+AcBnhB2K4kToiI3kTAWwzHhhRFN9o9yIzUGFW/k2skoj8trc3PuOt4QodCLnWCYgc9pTNqi+F6bglTeEKU1AaXSqeEKdkFD7FiI1cKaMYxuzTSnJwszA8EKMjW37s7qKK5sXm5EBd9Rdycj59BcRTKrOg5zHBLyLZDcSBzF188Q70xtVmOsSw2HFG+IChntWKYO8H5YNNY2uBD7LuJWSXZXWozSzBLK572iOuvamj9Ko6atqgk5Y182ZYVNAhgsMw8mmRoQFVmQwld9whJsXjgKzw/LorjAhSwogxW4oADi1F+oKWy0hh1h0EWDQIUoWwnHJv0LMg4JdGy28lbYjtQ+i1dguajYe6xFc/WRT93LoNMo4LwGf8A88V5Mm3llGY2YH2Z/GudNoWqRXRPeF6UFpYMtRnqGAXXKgzNNLt6R0JGUAwXLgTOsiE9Y3Qg8Cdc+pNWBrrKtxtxlb995449167660f/vamKUjKWsHw4VHBuMyXA/URE9A10O4V5UKTPu1mK4LcDD/4DNrtCkddJxMh4pxvkZ6dZ65e/7May7Y2GY76/GpMmwR7LUO7N53OBBAswgc1wYILNjBg42iAwLkRgTtEwIFhwFz7UREkPvivuBL24v75v0GB97wXesPd693/8GyY319bxT85HOn0qvwRLZfhtYo3/LA6bCUo6LfpZgl1I1FAhgMgkVYLYgN1b//d2AgTMCYQbCAYgXHDh13GHhA4ICJJyy8IFEaLAD3h4C5kAtFQqFQcurfWjaLV2PyRji9JUYXZsehBt9lhP33Ag==) format('woff2');
```

Comment:
I've created https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1124 to address this